src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -107,6 +107,12 @@
     private final String id;
 
     /**
+     * The "global" id of this conference, set by the controller (e.g. jicofo)
+     * as opposed to the bridge.
+     */
```

Comment:
What purpose does this new field have? Why is a 'global' identifier useful to have? In what sense is it "global"?